### ISSUE TITLE
Can't install v1.8 in fresh virtual environment

### ISSUE THREAD

**Issue by berquist** [cclib/cclib#1256]
```
/tmp
[eric@osmium]$ pyenv shell 3.10.12
/tmp
[eric@osmium]$ python -m venv ./venv_10
/tmp
[eric@osmium]$ source ./venv_10/bin/activate
/tmp
[eric@osmium]$ python -m pip install cclib
Collecting cclib
  Using cached cclib-1.7.2-py3-none-any.whl
Collecting periodictable
  Using cached periodictable-1.6.1-py2.py3-none-any.whl (752 kB)
Collecting numpy
  Using cached numpy-1.25.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (18.2 MB)
Collecting scipy>=1.2.0
  Using cached scipy-1.11.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (36.3 MB)
Collecting packaging>=19.0
  Using cached packaging-23.1-py3-none-any.whl (48 kB)
Collecting pyparsing
  Using cached pyparsing-3.1.1-py3-none-any.whl (103 kB)
Installing collected packages: pyparsing, packaging, numpy, scipy, periodictable, cclib
Successfully installed cclib-1.7.2 numpy-1.25.2 packaging-23.1 periodictable-1.6.1 pyparsing-3.1.1 scipy-1.11.2

[notice] A new release of pip is available: 23.0.1 -> 23.2.1
[notice] To update, run: pip install --upgrade pip
/tmp
[eric@osmium]$ python -m pip list
Package       Version
------------- -------
cclib         1.7.2
numpy         1.25.2
packaging     23.1
periodictable 1.6.1
pip           23.0.1
pyparsing     3.1.1
scipy         1.11.2
setuptools    65.5.0

[notice] A new release of pip is available: 23.0.1 -> 23.2.1
[notice] To update, run: pip install --upgrade pip
/tmp
[eric@osmium]$ python
Python 3.10.12 (main, Aug 20 2023, 23:52:47) [GCC 13.2.1 20230801] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import cclib
>>>
/tmp
[eric@osmium]$ python -m pip install -i https://test.pypi.org/simple/ --no-deps -U cclib
Looking in indexes: https://test.pypi.org/simple/
Requirement already satisfied: cclib in ./venv_10/lib/python3.10/site-packages (1.7.2)
Collecting cclib
  Using cached https://test-files.pythonhosted.org/packages/a3/a2/25d813596a1c7e4c33ebd8e2bba1563529a42a1c34ae4796ec2e71d2a2ed/cclib-1.8-py3-none-any.whl (346 kB)
Installing collected packages: cclib
  Attempting uninstall: cclib
    Found existing installation: cclib 1.7.2
    Uninstalling cclib-1.7.2:
      Successfully uninstalled cclib-1.7.2
Successfully installed cclib-1.8

[notice] A new release of pip is available: 23.0.1 -> 23.2.1
[notice] To update, run: pip install --upgrade pip
/tmp
[eric@osmium]$ python
Python 3.10.12 (main, Aug 20 2023, 23:52:47) [GCC 13.2.1 20230801] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import cclib
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/venv_10/lib/python3.10/site-packages/cclib/__init__.py", line 30, in <module>
    from cclib import parser
  File "/tmp/venv_10/lib/python3.10/site-packages/cclib/parser/__init__.py", line 17, in <module>
    from cclib.parser.adfparser import ADF
  File "/tmp/venv_10/lib/python3.10/site-packages/cclib/parser/adfparser.py", line 15, in <module>
    from cclib.parser import logfileparser
  File "/tmp/venv_10/lib/python3.10/site-packages/cclib/parser/logfileparser.py", line 19, in <module>
    from cclib.parser import utils
  File "/tmp/venv_10/lib/python3.10/site-packages/cclib/parser/utils.py", line 29, in <module>
    _found_scipy = find_package("scipy")
  File "/tmp/venv_10/lib/python3.10/site-packages/cclib/parser/utils.py", line 25, in find_package
    module_spec = importlib.util.find_spec(package)
AttributeError: module 'importlib' has no attribute 'util'
```

**Comment by berquist** [comment]
It happens in venvs created from pyenv as well as the system Python, but the full system Python works
```
$ md5sum /home/eric/.pyenv/versions/3.10.12/lib/python3.10/importlib/__init__.py /usr/lib/python3.11/importlib/__init__.py
8f5053714b9feff971f456e6ee52a274  /home/eric/.pyenv/versions/3.10.12/lib/python3.10/importlib/__init__.py
8f5053714b9feff971f456e6ee52a274  /usr/lib/python3.11/importlib/__init__.py
```
```
$ pyenv which python
/usr/bin/python
$ python
Python 3.11.3 (main, Jun  5 2023, 09:32:32) [GCC 13.1.1 20230429] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import importlib
>>> importlib.util
<module 'importlib.util' (frozen)>
```

**Comment by berquist** [comment]
https://discuss.python.org/t/python3-11-importlib-no-longer-exposes-util/25641

**Comment by shivupa** [comment]
I can reproduce this in the venv

**Comment by shivupa** [comment]
closed via #1259 ?

**Comment by berquist** [comment]
Yes
